OpenOversight/tests/routes/test_officer_and_department.py

```diff
@@ -114,15 +119,46 @@ def test_admin_can_add_officer_badge_number(mockdata, client, session):
         )
 
         assert 'Added new assignment' in rv.data.decode('utf-8')
+        assert '2019-01-01' in rv.data.decode('utf-8')
+        assert '2019-12-31' in rv.data.decode('utf-8')
```

Comment:
Yea I definitely agree testing the db is also necessary. I think it makes sense to keep the `assert '2019-01-01' in rv.data.decode('utf-8')` in as well, since that's effectively testing the template code, which was also changed in this PR. Do you agree?

Reply:
I agree, good point